src/tsbench/forecasts/prediction.py

```diff
@@ -20,7 +20,11 @@
 from gluonts.evaluation.backtest import make_evaluation_predictions
 from gluonts.model.forecast import QuantileForecast, SampleForecast
 from gluonts.model.predictor import ParallelizedPredictor, Predictor
-from gluonts.support.util import maybe_len
+try:
+    # TODO this problem may be caused by the version of gluonts?
+    from gluonts.support.util import maybe_len
+except:
+    from gluonts.itertools import maybe_len
```

Comment:
If the Python version is fixed, only one should be necessary.

Reply:
ok, I will fix the version of gluon-ts and remove redundancy code